config/samples/kustomization.yaml

```diff
@@ -6,7 +6,6 @@ resources:
 - downloadrequest.yaml
 - podvolumebackup.yaml
 - podvolumerestore.yaml
-- resticrepository.yaml
```

Comment:
Shouldn't we keep this for backwards compatibility ?

Reply:
I believe there is no need, will test to confirm

But worst case scenario (upgrade), what I think will happen is: on operator UI, `resticrepository` will not be shown, but CRD will not be deleted from cluster